src/rpc/mining.cpp

```diff
@@ -719,19 +719,15 @@ class submitblock_StateCatcher : public CValidationInterface
 
 UniValue submitblock(const JSONRPCRequest& request)
 {
+    // We allow 2 arguments for compliance with BIP22. Argument 2 is ignored.
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
         throw std::runtime_error(
-            "submitblock \"hexdata\" ( \"jsonparametersobject\" )\n"
+            "submitblock \"hexdata\"\n"
             "\nAttempts to submit new block to network.\n"
-            "The 'jsonparametersobject' parameter is currently ignored.\n"
```

Comment:
I think this line should be kept, as well as the `(\"jsonparametersobject\")` above. 

Removing the details of the parameters below is fine with me if they're all ignored, but just state that exlicitly.
```
2. \"parameters\"     (string, optional) object of optional parameters (currently ignored, but accepted for BIP22 compatbility)
```